app/Http/Controllers/Legacy/Web/ActionsController.php

```diff
@@ -84,10 +85,16 @@ public function store(Request $request)
      */
     public function update(Request $request, Action $action)
     {
+        Validator::make($request, [
+            'callpower_campaign_id' => [
+                'required_if:post_type,phone-call',
+                Rule::unique('actions')->ignore($action),
+            ],
+        ]);
+
```

Comment:
@DFurnes is this the correct way to do this?

Reply:
Does it work when you test it out? It looks like this is instantiating a validator, but not using it (which you'd do by calling `->validate()` on the returned "validator" object, or conditionally running code on `->fails()`).

You can group this with the rest of our validation rules in the `$this->validate(…)` call below:

```php
$this->validate($request, [
    // ...
    'callpower_campaign_id' => ['required_if:post_type,phone-call', Rule::unique('actions')->ignore($action)],
```

Reply:
@DFurnes got this to work! I had to include `$action->id` instead of just `$action` because for some reason it seemed to be grabbing the `campaign_id` here.